src/main/scala/mesosphere/marathon/Features.scala

```diff
@@ -11,31 +11,28 @@ object Features {
   //enable external volumes
   lazy val EXTERNAL_VOLUMES = "external_volumes"
 
-  //enable secrets
-  lazy val SECRETS = "secrets"
-
   //enable GPUs
   lazy val GPU_RESOURCES = "gpu_resources"
 
-  //enable maintenance mode
-  lazy val MAINTENANCE_MODE = "maintenance_mode"
```

Comment:
This PR doesn't say anything about getting rid of MAINTENANCE_MODE feature. Could you update the description?